### PR TITLE
update BR links

### PR DESCRIPTION
**Pull request by senhor-R**
update BR links and add channels

### CODE REVIEW

File: streams/br.m3u [hunk 1]
```diff
@@ -35,6 +35,10 @@ https://evpp.mm.uol.com.br/geob_band/band/playlist.m3u8
 #EXTINF:-1 tvg-id="BandNews.br" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0",Bandnews (720p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0
 https://evpp.mm.uol.com.br/geob_band/bandnewstv/playlist.m3u8
+#EXTINF:-1 tvg-id="BANDMG.br",BAND MG (720p)
+http://168.205.87.198:8555/live/1501/123456/19.m3u8
+#EXTINF:-1 tvg-id="BANDSP.br",BAND SP (720p)
+http://niptv.biz:25461/gustavohrocha@gmail.com/espelho/159
```
Comment (BellezaEmporium) [inline streams/br.m3u:41]: Same treatment in here.

File: streams/br.m3u [hunk 2]
```diff
@@ -91,8 +97,12 @@ https://stmv1.srvif.com/classiquetv1/classiquetv1/playlist.m3u8
 https://stmv1.srvif.com/classique/classique/playlist.m3u8
 #EXTINF:-1 tvg-id="ClassiqueTVWestern.br",Classique TV Western (360p)
 https://stmv1.srvif.com/classiquew/classiquew/playlist.m3u8
+#EXTINF:-1 tvg-id="CANALFUTURA.br",CANAL FUTURA
+http://168.205.87.198:8555/live/1501/123456/67.m3u8
+#EXTINF:-1 tvg-id="CANCAONOVA.br",CANÇAO NOVA
+http://168.205.87.198:8555/live/1501/123456/25.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:103]: Same treatment in here.

File: streams/br.m3u [hunk 3]
```diff
@@ -123,6 +133,20 @@ http://flash.softhost.com.br:1935/fonte/fontetv/live.m3u8
 https://stream.ichibantv.com:3764/hybrid/play.m3u8
 #EXTINF:-1 tvg-id="GhostTV.br",Ghost TV (712p) [Not 24/7]
 https://stmv.video.expressolider.com.br/ghostv/ghostv/playlist.m3u8
+#EXTINF:-1 tvg-id="GLOBOMG.br",GLOBO MG (1080p)
+http://168.205.87.198:8555/live/1501/123456/70.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:137]: ^

File: streams/br.m3u [hunk 4]
```diff
@@ -123,6 +133,20 @@ http://flash.softhost.com.br:1935/fonte/fontetv/live.m3u8
...
+#EXTINF:-1 tvg-id="GLOBONEWS.br",GLOBO NEWS (720p)
+http://143.137.216.38:8080/gmsolucoes/gm121109/229
+#EXTINF:-1 tvg-id="FLATV.br",FLA TV [NOT 24/7]
+https://ythls.onrender.com/channel/UCOa-WaNwQaoyFHLCDk7qKIw.m3u8
+#EXTINF:-1 tvg-id="GLOBORJ.br",GLOBO RJ (1080p)
+http://firp.me:80/live/TeP0zd/7mYuL7/381780.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:143]: ^

File: streams/br.m3u [hunk 5]
```diff
@@ -123,6 +133,20 @@ http://flash.softhost.com.br:1935/fonte/fontetv/live.m3u8
...
+#EXTINF:-1 tvg-id="GLNTV.br",GLN TV (720p)
+https://stmv1.paineltv.net/glnfilmes/glnfilmes/chunklist_w1263571980.m3u8
+#EXTINF:-1 tvg-id="GLOBOSP.br",GLOBO SP (1080p)
+http://acsc.cc:80/rg_advogados@outlook.com3/Tv452973/72040.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:147]: ^

File: streams/br.m3u [hunk 6]
```diff
@@ -135,6 +159,8 @@ https://cdn.jmvstream.com/w/LVW-9883/LVW9883_lFcfKysrHF/chunklist.m3u8
 https://cdn.jmvstream.com/w/LVW-8529/LVW8529_S8HhwGeAQU/chunklist.m3u8
 #EXTINF:-1 tvg-id="JovemPanNews.br",Jovem Pan News (JP News) (1080p) [Not 24/7]
 https://d6yfbj4xxtrod.cloudfront.net/out/v1/7836eb391ec24452b149f3dc6df15bbd/index.m3u8
+#EXTINF:-1 tvg-id="KIDSCHANNEL.br",KIDS CHANNEL
+http://168.205.87.198:8555/live/1501/123456/143.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:163]: ^

File: streams/br.m3u [hunk 7]
```diff
@@ -195,14 +221,14 @@ https://playplusbsa-lh.akamaihd.net/i/pp_bsa@377860/index_720_av-p.m3u8
 https://playplusgoya-lh.akamaihd.net/i/pp_gna@377833/index_720_av-p.m3u8
 #EXTINF:-1 tvg-id="RecordTVItapoan.br",Record TV Itapoan (720p) [Geo-blocked]
 https://playplussdr-lh.akamaihd.net/i/pp_sdr@377858/index_720_av-b.m3u8
-#EXTINF:-1 tvg-id="RecordTVMinas.br",Record TV Minas (720p) [Geo-blocked]
-https://playplusbh-lh.akamaihd.net/i/pp_bh@377862/index_720_av-p.m3u8
-#EXTINF:-1 tvg-id="RecordTVRio.br",Record TV Rio (720p) [Geo-blocked]
-https://playplusrjo-lh.akamaihd.net/i/pp_rj@377859/index_720_av-p.m3u8
+#EXTINF:-1 tvg-id="RecordTVMinas.br",Record TV Minas (720p)
+http://168.205.87.198:8555/live/1501/123456/96.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:225]: ^

File: streams/br.m3u [hunk 8]
```diff
@@ -195,14 +221,14 @@ https://playplusbsa-lh.akamaihd.net/i/pp_bsa@377860/index_720_av-p.m3u8
...
+#EXTINF:-1 tvg-id="RecordTVRio.br",Record TV Rio (720p)
+https://iptv.vivatele.com:8443/live/viva_padrao/viva2384/3.ts
```
Comment (BellezaEmporium) [inline streams/br.m3u:227]: ^

File: streams/br.m3u [hunk 9]
```diff
@@ -195,14 +221,14 @@ https://playplusbsa-lh.akamaihd.net/i/pp_bsa@377860/index_720_av-p.m3u8
...
 #EXTINF:-1 tvg-id="RecordTVRS.br",Record TV RS (720p) [Geo-blocked]
 https://playpluspoa-lh.akamaihd.net/i/pp_poa@377864/index_720_av-p.m3u8
-#EXTINF:-1 tvg-id="RecordTVSaoPaulo.br",Record TV SP (720p) [Geo-blocked]
-https://playplusspo-lh.akamaihd.net/i/pp_sp@350176/index_720_av-p.m3u8
+#EXTINF:-1 tvg-id="RecordTVSaoPaulo.br",Record TV SP (720p)
+http://143.137.216.38:8080/gmsolucoes/gm121109/451
```
Comment (BellezaEmporium) [inline streams/br.m3u:231]: ^

File: streams/br.m3u [hunk 10]
```diff
@@ -249,6 +279,8 @@ https://59f1cbe63db89.streamlock.net:1443/tvkidmais/_definst_/tvkidmais/playlist
 http://tv02.logicahost.com.br:1935/rederc/rederc/live.m3u8
 #EXTINF:-1 tvg-id="RedeSeculo21.br",Rede Século 21 (1080p) [Not 24/7]
 https://cdn.jmvstream.com/w/LVW-10024/ngrp:LVW10024_H3QLdAY6kx_all/playlist.m3u8
+#EXTINF:-1 tvg-id="REDESUPER.br",REDE SUPER (720p)
+http://168.205.87.198:8555/live/1501/123456/163.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:283]: ^

File: streams/br.m3u [hunk 11]
```diff
@@ -565,6 +605,8 @@ https://streamer1.streamhost.org/salive/GMI3anjoh/playlist.m3u8
 https://cdn-grupo-10049.ciclano.io:1443/grupo-10049/grupo-10049/playlist.m3u8
 #EXTINF:-1 tvg-id="TVTropical.br",TV Tropical (480p)
 https://59f2354c05961.streamlock.net:1443/portaldatropical/_definst_/portaldatropical/playlist.m3u8
+#EXTINF:-1 tvg-id="TVTURFE.br",TV TURFE (480p)
+http://168.205.87.198:8555/live/1501/123456/152.m3u8
```
Comment (BellezaEmporium) [inline streams/br.m3u:609]: ^

**Review by freearhey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**